src/foam/u2/wizard/debug/scenarios.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.debug.scenarios',
+  name: 'MinMaxChoicePrereqLiftScenario',
```

Comment:
Why is the file named after the package instead of the model?

Reply:
easier to add more scenarios - I could also see metaprogramming being used in here to create a number of similar scenarios